src/main/java/mate/academy/Main.java

```diff
@@ -2,17 +2,23 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
```

Comment:
Ensure that `RegistrationException` and `AuthenticationException` do not extend `RuntimeException`. This is a significant deviation from the task requirements.